lib/AvatarGroup/styles.scss

```diff
@@ -0,0 +1,3 @@
+.avatar-group-list {
+  margin-right: 1em;
```

Comment:
The reason I used `1em` is because that is the value that we loose here https://github.com/gathercontent/gather-ui/blob/63a5085884ac582479dc048b0189033639108c3c/lib/Avatar/styles.scss#L93-L99
when the avatars are in a group - this doesn't add any extra margin around the group, it just resets the margin that is lost, if that makes sense 😄